packages/auth-construct/src/types.ts

```diff
@@ -139,8 +139,19 @@ export type FacebookProviderProps = Omit<
  */
 export type OidcProviderProps = Omit<
   cognito.UserPoolIdentityProviderOidcProps,
-  'userPool'
->;
+  'userPool' | 'attributeRequestMethod'
+> & {
```

Comment:
maybe instead of fixing cdk types we should start investing in our types + mapping?
this call is easier to make pre-GA.

Reply:
Originally did this to reduce maintenance overhead, we save a lot on types by doing this specially for oauth